debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/rest/DebeziumMySqlConnectorResource.java

```diff
@@ -42,6 +53,40 @@ public Connector getConnector() {
         return new MySqlConnector();
     }
 
+    @Override
+    public List<DataCollection> getMatchingCollections(Configuration configuration) {
+        final MySqlConnectorConfig config = new MySqlConnectorConfig(configuration);
+        final MySqlConnection.MySqlConnectionConfiguration connectionConfig = new MySqlConnection.MySqlConnectionConfiguration(configuration);
+        try (MySqlConnection connection = new MySqlConnection(connectionConfig)) {
+            Set<TableId> tables;
+
+            final List<String> databaseNames = new ArrayList<>();
+            connection.query("SHOW DATABASES", rs -> {
```

Comment:
It might be a good idea to move this into the connector code.

Reply:
True. Could you please file a Jira for this? Thank you

Reply:
@rk3rn3r This is vry similar comment to https://github.com/debezium/debezium/pull/4985#issuecomment-1801586596

I am fine with handling this as a separate PR but quite fastr after this one to make sure we avoid any divergence. Could you please create a Jira for it and chedule it for the next release?